graf3d/eve7/src/REveDataSimpleProxyBuilder.cxx

```diff
@@ -301,7 +301,7 @@ REveElement *REveCollectionCompound::GetSelectionMaster()
    fCollection->GetItemList()->RefSelectedSet().clear();
    try {
 
-      std::size_t found = fName.find_last_of(" ");
+      std::size_t found = fName.find_last_of(' ');
```

Comment:
Here is not clear if such change is equivalent. It will use other signature for `std::string::find_last_of` method.

Reply:
Thanks for your comment! As far as I understand from the clang-tidy docs, the behavior is equivalent but the overload that takes a single character is more efficient:
https://releases.llvm.org/11.0.0/tools/clang/tools/extra/docs/clang-tidy/checks/performance-faster-string-find.html

See also:
https://cplusplus.com/reference/string/string/find_last_of/